components/automate-ui/src/app/modules/infra-proxy/chef-servers-list/chef-servers-list.component.spec.ts

```diff
@@ -208,5 +213,56 @@ describe('ChefServersListComponent', () => {
 
     });
   });
+
+  describe('delete modal', () => {
+    const mockChefKeyEvent = new KeyboardEvent('keypress') as ChefKeyboardEvent;
```

Comment:
I recently noticed this is the wrong usage and have started a PR that will be fixing those that already exist. The problem is we should be using `MatOptionSelectionChange` rather than `ChefKeyboardEvent`. You can see the errors when we tighten up linting by changing `strictOutputEventTypes` to true in tsconfig.json, then running `ng build`. 
So please change to this here and if you use it elsewhere:
```suggestion
    const mockEvent = { isUserInput: true } as MatOptionSelectionChange;
```